lib/kazoo/replica_reassigner.rb

```diff
@@ -0,0 +1,51 @@
+module Kazoo
+
+  # Helper class to reassign replicas to brokers.
+  class ReplicaReassigner
+    attr_reader :cluster
+
+    def initialize(cluster)
+      @cluster = cluster
+    end
+
+    def self.agony(from, to)
+      (to - from).length
+    end
+
+    def self.valid_replicaset?(replicas, minimum_replicas: 1)
+      replicas == replicas.uniq && replicas.length >= minimum_replicas
+    end
+
+    def self.safe_reassignment?(from, to, max_agony: 1, minimum_replicas: 1)
```

Comment:
'agony' is a cute name but not very descriptive; I'd remove it entirely and just hardcode it to 1, since it seems anything else causes lag?